.github/ISSUE_TEMPLATE/extensibility-request.yaml

```diff
@@ -5,24 +5,24 @@ body:
   - type: markdown
     attributes:
       value: |
-        Thanks for taking the time to fill out this extensibility request form. 
+        Thanks for submitting an extensibility request.
 
-        ⚠️⚠️ BEFORE WE GET STARTED ⚠️⚠️
+        ⚠️⚠️ Before you start ⚠️⚠️
         Before you create a new issue, please check the following:
-        🔎 Search existing issues to avoid creating duplicates.
+        🔎 Search the existing issues to avoid a duplicate.
   - type: textarea
     id: describe-the-request
     attributes:
       label: Describe the request
-      description: A clear and concise description of what the extensibility request is, which you want Microsoft to address
+      description: Provide a clear, concise description of what you need. Tell us what you want to address.
       placeholder: Describe the extensibility request
     validations:
       required: true
   - type: textarea
     id: additional-context
     attributes:
       label: Additional context
-      description: Explain why you need this request to be fulfilled. This will help Microsoft assess, if your request could get solved in other ways
-      placeholder: Explain why you need this request
+      description: Explain the benefits of what you’re requesting. Help us assess the best way to fulfil your need.
```

Comment:
Explain the benefits of what you’re requesting. Help us assess the best way to fulfill your need.